src/test/java/com/example/onlinebookstore/OnlineBookStoreApplicationTests.java

```diff
@@ -1,11 +1,16 @@
 package com.example.onlinebookstore;
 
+import com.example.onlinebookstore.security.JwtUtil;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
 
 @SpringBootTest
 class OnlineBookStoreApplicationTests {
 
+    @MockBean
+    private JwtUtil jwtUtil;
+
```

Comment:
Why do you need it here?

Reply:
to avoid this issue: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'securityConfig' defined in file [/Users/nazar5n/IdeaProjects/online-book-store/target/classes/com/example/onlinebookstore/config/SecurityConfig.class]: Unsatisfied dependency expressed through constructor parameter 1: Error creating bean with name 'jwtAuthenticationFilter' defined in file [/Users/nazar5n/IdeaProjects/online-book-store/target/classes/com/example/onlinebookstore/security/JwtAuthenticationFilter.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'jwtUtil' defined in file [/Users/nazar5n/IdeaProjects/online-book-store/target/classes/com/example/onlinebookstore/security/JwtUtil.class]: Unexpected exception during bean creation